From 76eae12fd08ffdf5ce04e2802783cace57c7d99d Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Fri, 28 Aug 2020 12:11:33 +0800
Subject: [PATCH] rtc: Kconfig: Remove RTC_DRV_IMX_SC dependent option IMX_SCU

Since the 4.19 SDK did not put the message unit under the mailbox, but
A separately created IPC interface, so the IMX_SCU interface is not
opened, so there is no need to rely on the IMX_SCU option

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 drivers/rtc/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/rtc/Kconfig b/drivers/rtc/Kconfig
index cbca50027fe2..39c61a203823 100644
--- a/drivers/rtc/Kconfig
+++ b/drivers/rtc/Kconfig
@@ -1737,7 +1737,6 @@ config RTC_DRV_SNVS
 	   will be called "rtc-snvs".
 
 config RTC_DRV_IMX_SC
-	depends on IMX_SCU
 	depends on HAVE_ARM_SMCCC
 	tristate "NXP i.MX System Controller RTC support"
 	help
-- 
2.17.1

